games-engines/fizmo/fizmo-0.7.10.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A Z-Machine interpreter development library"
+DESCRIPTION="That means it allows you to play interactive fiction, also \
+known as textadventures, which were implemented either by Infocom or created \
+using the Inform compiler on a terminal, or to develop your own interactive \
+fiction front end in C or any C-related language like C++ or Objective-C."
+HOMEPAGE="https://christoph-ender.de/fizmo/"
+COPYRIGHT="2009–2014 Christoph Ender"
+LICENSE="Fizmo"
+REVISION="1"
+SOURCE_URI="https://christoph-ender.de/fizmo/source/fizmo-$portVersion.tar.gz"
+CHECKSUM_SHA256="0e3561492ece58ff60eba768f3b2cfa943ba111736b0f63b775e3face590462b"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fizmo$secondaryArchSuffix = $portVersion
+	cmd:fizmo_console$secondaryArchSuffix = $portVersion
+	cmd:fizmo_ncursesw$secondaryArchSuffix = $portVersion
+	"
+
```

Comment:
The [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering) say we don't need a blank line between PROVIDES and REQUIRES.
Same rule between PROVIDES_devel and REQUIRES_devel..
Same rule between BUILD_REQUIRES and  BUILD_PREREQUIRES.

There seems to be an exception, however, when there are conditional additions to PROVIDES or PROVIDES_devel, in which cases the conditional addition is followed by a blank line.
Well, this is what I have noticed, but I don't know if it is written somewhere.